cloud-gateway-package/src/main/resources/bin/start.sh

```diff
@@ -125,6 +125,7 @@ _BPX_JOBNAME=${ZWE_zowe_job_prefix}${CLOUD_GATEWAY_CODE} java \
     -Djava.io.tmpdir=${TMPDIR:-/tmp} \
     -Dspring.profiles.active=${ZWE_configs_spring_profiles_active:-} \
     -Dspring.profiles.include=$LOG_LEVEL \
+    -Dapiml.security.x509.registry.allowedUsers=${ZWE_configs_apiml_security_x509_registry_allowedUsers} \
```

Comment:
```suggestion
    -Dapiml.security.x509.registry.allowedUsers=${ZWE_configs_apiml_security_x509_registry_allowedUsers:-} \
```
I am not sure, but I guess there are two issues:
1. the valid is not required
2. default value if null (see `org.zowe.apiml.cloudgatewayservice.config.WebSecurity#allowedUsers`) and empty string is not `null`